scene/gui/color_picker.cpp

```diff
@@ -691,31 +695,39 @@ ColorPicker::ColorPicker() :
 	}
 
 	HBoxContainer *hhb = memnew(HBoxContainer);
+	vbr->add_child(hhb);
 
 	btn_mode = memnew(CheckButton);
+	hhb->add_child(btn_mode);
 	btn_mode->set_text(TTR("Raw Mode"));
 	btn_mode->connect("toggled", this, "set_raw_mode");
-	hhb->add_child(btn_mode);
-	vbr->add_child(hhb);
+
 	text_type = memnew(Button);
-	text_type->set_flat(true);
-	text_type->connect("pressed", this, "_text_type_toggled");
 	hhb->add_child(text_type);
+	text_type->set_text("#");
+	text_type->set_tooltip(TTR("Switch between hexadecimal and code values."));
+	if (Engine::get_singleton()->is_editor_hint()) {
+		text_type->set_custom_minimum_size(Size2(28 * EDSCALE, 0)); // Adjust for the width of the "Script" icon.
```

Comment:
`EDSCALE` is only available within `#ifdef TOOLS_ENABLED`.

Reply:
Of course, fixed it.